src/openapi-generator.ts

```diff
@@ -727,6 +727,13 @@ export class OpenAPIGenerator {
         return;
       }
 
+      if (isZodType(value, 'ZodNativeEnum')) {
+        Object.values(value._def.values).forEach((enumValue: string) => {
```

Comment:
Why do we need Object.values here as opposed to the direct usage of `value._def.values` above?